### ISSUE TITLE
Have you tought of making a Kickstarter campaign for making free assets for FreeHL & CS?

### ISSUE THREAD

**Issue by xplshn** [eukara/FreeCS#26]
Hey, I think this project is awesome, I think more people should know about it and I had an idea. You know OpenARENA? A clone of Quake, based on Quake's engine, same physics, same engine, same gameplay, new assets, it'd be awesome if FreeCS & FreeHL had financing to make free assets. Tell me what you think about it, also, if you don't like the idea, could you guide me in the right direction, so I can try to?

**Comment by eukara** [comment]
Yeah, ever since the project's inception in 2016 someone came around asking about it. I very dearly welcome it and even suggested it in the issue here: https://github.com/eukara/freecs/issues/23

The problem is that while I am very good at cloning game logic, I am not qualified to make replacement content by myself (nor do I have the time to do so). Making a campaign is also a lot of work. I don't know if I have the energy doing that when I'm struggling right now with everything else in life.

This is where anyone else can come in and contribute, although it should be a separate project by itself so if people want to use mine to play it with the original data, they can.

You can make models using any of the formats supported by FTEQW, and levels with any of the available Quake BSP ones.
I'd highly suggest not using the Half-Life formats though, as they're restricted by the LICENSE of the original Half-Life SDK (you'll undoubtedly be using Valve's tools to create those, so that can't happen).

**Comment by xplshn** [comment]
Okay, I'm not a map modder, but I can try, if I come up with something useful, I'll make you know

**Comment by getgle** [comment]
Yeah this game would explode with popularity if it had free assets packed in like openarena

**Comment by xplshn** [comment]
Yup, it has been done before, with FreeDOOM, LibreQuake, OpenARENA, Xonotic, Unvanquished, Red Eclipse, etc.
